utoipa-gen/src/schema/component.rs

```diff
@@ -456,72 +576,53 @@ impl ToTokens for ComplexEnum<'_> {
             );
         }
 
-        let capasity = self.variants.len();
-        tokens.extend(quote! {
-            Into::<utoipa::openapi::schema::OneOfBuilder>::into(utoipa::openapi::OneOf::with_capacity(#capasity))
-        });
+        let capacity = self.variants.len();
+        tokens.extend(quote! {});
```

Comment:
Useless tokens.extend(..) :)
```suggestion
```

Reply:
Haha not sure how this one got in there!